mysql-test/suite/galera/t/MW-258.test

```diff
@@ -13,6 +15,7 @@ SHOW VARIABLES LIKE 'wsrep_desync';
 --connection node_1a
 SET SESSION wsrep_sync_wait = 0;
 SET SESSION wsrep_osu_method = RSU;
+SET GLOBAL wsrep_provider_options = 'dbug=d,wsrep_desync_left_local_monitor';
```

Comment:
We should use the script instead
--let $galera_sync_point = apply_monitor_slave_enter_sync
--source include/galera_set_sync_point.inc